### PR TITLE
Ensure correct params are passed for CI/CD km mt stress test jobs

### PR DESCRIPTION
**Pull request by dv-msft**
## Description

Fixes runtime parameter handling to ensure correct and exclusive execution of the scheduled km stress tests. 

## Testing

CI/CD Scheduled Run

## Documentation

No doc changes.

## Installation

No installation changes.

Fixes: #2771 
Fixes: #2772

### CODE REVIEW

**Review by Alan-Jowett — DISMISSED** [review]

File: scripts/execute_ebpf_cicd_tests.ps1 [hunk 1]
```diff
@@ -33,12 +33,17 @@ foreach ($VM in $VMList) {
         -RestartExtension $RestartExtension
 }
 
-# Run XDP Tests.
-Invoke-XDPTestsOnVM $Config.Interfaces $VMList[0].Name
+if ($RunKmStressTests -eq $false) {
```
Comment (dthaler) [inline scripts/execute_ebpf_cicd_tests.ps1:36]: why $false instead of $true?

Comment (dv-msft) [reply scripts/execute_ebpf_cicd_tests.ps1:36]: The km stress tests are scheduled runs and I've extended this (existing) script to handle those tests as well.  I'd previously added a separate script for the km stress tests, but a prior feedback recommended that we do this instead.

The `$RunKmStressTests` parameter ensures that when set to true, _only_ the (new) stress tests are run. The existing tests, i.e, `Invoke-XDPTestsOnVM`, `Invoke-ConnectRedirectTestsOnVM` and `Invoke-ConnectRedirectTestsOnVM` are already handled by other jobs, so re-executing them again along with the stress tests would be redundant.

Comment (dthaler) [reply scripts/execute_ebpf_cicd_tests.ps1:36]: The name `RunKmStressTests` does not imply that "only" the stress tests are run, it implies that the stress tests are run in addition.  Either change the name or change the object.  E.g., if it _disables_ running non-stress tests, then it should be phrased as such.

Comment (dv-msft) [reply scripts/execute_ebpf_cicd_tests.ps1]: Fixed. Renamed `RunKmStressTests` to `RunKmStressTestsOnly`.

**Review by dthaler — APPROVED** [review]

**Review by Alan-Jowett — APPROVED** [review]

**Pull request merged**